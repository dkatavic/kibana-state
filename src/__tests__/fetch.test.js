const { fetch } = require('../fetch')

test('Fetch current kibana state', async () => {
  const consoleLogSpy = jest.fn()
  console.log = consoleLogSpy
  await fetch({
    kibanaIndexName: '.kibana'
  })
  expect(consoleLogSpy.mock.calls[0][0]).toMatchSnapshot()
})

test('Fetch current kibana state with prettyPrint', async () => {
  const consoleLogSpy = jest.fn()
  console.log = consoleLogSpy
  await fetch({
    kibanaIndexName: '.kibana',
    prettyPrint: true
  })
  expect(consoleLogSpy.mock.calls[0][0]).toMatchSnapshot()
})
