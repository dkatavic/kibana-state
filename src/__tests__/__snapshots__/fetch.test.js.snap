// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fetch current kibana state 1`] = `"[{\\"_index\\":\\".kibana\\",\\"_type\\":\\"doc\\",\\"_id\\":\\"config:6.2.2\\",\\"_score\\":1,\\"_source\\":{\\"type\\":\\"config\\",\\"updated_at\\":\\"2018-04-25T08:56:45.446Z\\",\\"config\\":{\\"buildNum\\":16588,\\"defaultIndex\\":\\"a7480050-3118-11e8-b629-33a5d6f661b4\\",\\"tr-version\\":\\"1.0.1\\"}},\\"config\\":{\\"tr-checksum\\":\\"ec55486b9d2841da0d6f7bd908225265defe695a\\"}},{\\"_index\\":\\".kibana\\",\\"_type\\":\\"doc\\",\\"_id\\":\\"dashboard:host_dashboard\\",\\"_score\\":1,\\"_source\\":{\\"config\\":{\\"tr-checksum\\":\\"00d8211f109fd19f9d84a39523b9bdd716b8c3b5\\"},\\"type\\":\\"dashboard\\",\\"updated_at\\":\\"2018-04-24T22:21:35.904Z\\",\\"dashboard\\":{\\"title\\":\\"Host Dashboard\\",\\"hits\\":0,\\"description\\":\\"\\",\\"panelsJSON\\":\\"[{\\\\\\"panelIndex\\\\\\":\\\\\\"4\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"4\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"fca79400-44cf-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"5\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"5\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"178f61d0-47dc-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"6\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"6\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"61b18a10-44d8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"7\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"7\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"6e3552d0-47f8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"8\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":6,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"8\\\\\\"},\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"id\\\\\\":\\\\\\"04535f20-4801-11e8-8d68-419cfa041388\\\\\\"}]\\",\\"optionsJSON\\":\\"{\\\\\\"darkTheme\\\\\\":false,\\\\\\"hidePanelTitles\\\\\\":false,\\\\\\"useMargins\\\\\\":true}\\",\\"version\\":1,\\"timeRestore\\":false,\\"kibanaSavedObjectMeta\\":{\\"searchSourceJSON\\":\\"{\\\\\\"query\\\\\\":{\\\\\\"language\\\\\\":\\\\\\"lucene\\\\\\",\\\\\\"query\\\\\\":\\\\\\"\\\\\\"},\\\\\\"filter\\\\\\":[],\\\\\\"highlightAll\\\\\\":true,\\\\\\"version\\\\\\":true}\\"}}},\\"config\\":{\\"tr-checksum\\":\\"5262fe31dbfa40490add6aaa32e54d846fac0f3c\\"}},{\\"_index\\":\\".kibana\\",\\"_type\\":\\"doc\\",\\"_id\\":\\"dashboard:3999b800-44cd-11e8-8d68-419cfa041388\\",\\"_score\\":1,\\"_source\\":{\\"config\\":{\\"tr-checksum\\":\\"499dbfd06272af8f92a0ae164c92c8da4c2efaaf\\"},\\"type\\":\\"dashboard\\",\\"updated_at\\":\\"2018-04-25T21:56:18.359Z\\",\\"dashboard\\":{\\"title\\":\\"Robin's Dashboard\\",\\"hits\\":0,\\"description\\":\\"\\",\\"panelsJSON\\":\\"[{\\\\\\"panelIndex\\\\\\":\\\\\\"1\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":6,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"1\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"61b18a10-44d8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"2\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":9,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"2\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"fca79400-44cf-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"3\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":12,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"3\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"6e3552d0-47f8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"4\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":12,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"4\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"178f61d0-47dc-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"5\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":15,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"5\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"04535f20-4801-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"6\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"6\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"00ea4aa0-489d-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"7\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"7\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"d9ac58a0-48a3-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"8\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"8\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"59da5c10-48a0-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"9\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"9\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"1a792fb0-48aa-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"10\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":6,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"10\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"5740c040-48b2-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"11\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":9,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"11\\\\\\"},\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"id\\\\\\":\\\\\\"311d6b30-48a7-11e8-8d68-419cfa041388\\\\\\"}]\\",\\"optionsJSON\\":\\"{\\\\\\"darkTheme\\\\\\":true,\\\\\\"hidePanelTitles\\\\\\":false,\\\\\\"useMargins\\\\\\":true}\\",\\"version\\":1,\\"timeRestore\\":false,\\"kibanaSavedObjectMeta\\":{\\"searchSourceJSON\\":\\"{\\\\\\"query\\\\\\":{\\\\\\"language\\\\\\":\\\\\\"lucene\\\\\\",\\\\\\"query\\\\\\":\\\\\\"\\\\\\"},\\\\\\"filter\\\\\\":[],\\\\\\"highlightAll\\\\\\":true,\\\\\\"version\\\\\\":true}\\"}}},\\"config\\":{\\"tr-checksum\\":\\"6d7d7f641171293dcdb118b7eecedb47ca7c5db5\\"}}]"`;

exports[`Fetch current kibana state with prettyPrint 1`] = `
"[
  {
    \\"_index\\": \\".kibana\\",
    \\"_type\\": \\"doc\\",
    \\"_id\\": \\"config:6.2.2\\",
    \\"_score\\": 1,
    \\"_source\\": {
      \\"type\\": \\"config\\",
      \\"updated_at\\": \\"2018-04-25T08:56:45.446Z\\",
      \\"config\\": {
        \\"buildNum\\": 16588,
        \\"defaultIndex\\": \\"a7480050-3118-11e8-b629-33a5d6f661b4\\",
        \\"tr-version\\": \\"1.0.1\\"
      }
    },
    \\"config\\": {
      \\"tr-checksum\\": \\"bfc0b31119f4d33682374a0ba7f6bdeb42048365\\"
    }
  },
  {
    \\"_index\\": \\".kibana\\",
    \\"_type\\": \\"doc\\",
    \\"_id\\": \\"dashboard:host_dashboard\\",
    \\"_score\\": 1,
    \\"_source\\": {
      \\"config\\": {
        \\"tr-checksum\\": \\"00d8211f109fd19f9d84a39523b9bdd716b8c3b5\\"
      },
      \\"type\\": \\"dashboard\\",
      \\"updated_at\\": \\"2018-04-24T22:21:35.904Z\\",
      \\"dashboard\\": {
        \\"title\\": \\"Host Dashboard\\",
        \\"hits\\": 0,
        \\"description\\": \\"\\",
        \\"panelsJSON\\": \\"[{\\\\\\"panelIndex\\\\\\":\\\\\\"4\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"4\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"fca79400-44cf-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"5\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"5\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"178f61d0-47dc-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"6\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"6\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"61b18a10-44d8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"7\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"7\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"6e3552d0-47f8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"8\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":6,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"8\\\\\\"},\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"id\\\\\\":\\\\\\"04535f20-4801-11e8-8d68-419cfa041388\\\\\\"}]\\",
        \\"optionsJSON\\": \\"{\\\\\\"darkTheme\\\\\\":false,\\\\\\"hidePanelTitles\\\\\\":false,\\\\\\"useMargins\\\\\\":true}\\",
        \\"version\\": 1,
        \\"timeRestore\\": false,
        \\"kibanaSavedObjectMeta\\": {
          \\"searchSourceJSON\\": \\"{\\\\\\"query\\\\\\":{\\\\\\"language\\\\\\":\\\\\\"lucene\\\\\\",\\\\\\"query\\\\\\":\\\\\\"\\\\\\"},\\\\\\"filter\\\\\\":[],\\\\\\"highlightAll\\\\\\":true,\\\\\\"version\\\\\\":true}\\"
        }
      }
    },
    \\"config\\": {
      \\"tr-checksum\\": \\"7ec01f29156e7bef9869d0aa1d713b76230670e5\\"
    }
  },
  {
    \\"_index\\": \\".kibana\\",
    \\"_type\\": \\"doc\\",
    \\"_id\\": \\"dashboard:3999b800-44cd-11e8-8d68-419cfa041388\\",
    \\"_score\\": 1,
    \\"_source\\": {
      \\"config\\": {
        \\"tr-checksum\\": \\"499dbfd06272af8f92a0ae164c92c8da4c2efaaf\\"
      },
      \\"type\\": \\"dashboard\\",
      \\"updated_at\\": \\"2018-04-25T21:56:18.359Z\\",
      \\"dashboard\\": {
        \\"title\\": \\"Robin's Dashboard\\",
        \\"hits\\": 0,
        \\"description\\": \\"\\",
        \\"panelsJSON\\": \\"[{\\\\\\"panelIndex\\\\\\":\\\\\\"1\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":6,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"1\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"61b18a10-44d8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"2\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":9,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"2\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"fca79400-44cf-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"3\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":12,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"3\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"6e3552d0-47f8-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"4\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":12,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"4\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"178f61d0-47dc-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"5\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":15,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"5\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"04535f20-4801-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"6\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"6\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"00ea4aa0-489d-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"7\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"7\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"d9ac58a0-48a3-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"8\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":0,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"8\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"59da5c10-48a0-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"9\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":6,\\\\\\"y\\\\\\":3,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"9\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"1a792fb0-48aa-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"10\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":6,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"10\\\\\\"},\\\\\\"id\\\\\\":\\\\\\"5740c040-48b2-11e8-8d68-419cfa041388\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\"},{\\\\\\"panelIndex\\\\\\":\\\\\\"11\\\\\\",\\\\\\"gridData\\\\\\":{\\\\\\"x\\\\\\":0,\\\\\\"y\\\\\\":9,\\\\\\"w\\\\\\":6,\\\\\\"h\\\\\\":3,\\\\\\"i\\\\\\":\\\\\\"11\\\\\\"},\\\\\\"version\\\\\\":\\\\\\"6.2.2\\\\\\",\\\\\\"type\\\\\\":\\\\\\"visualization\\\\\\",\\\\\\"id\\\\\\":\\\\\\"311d6b30-48a7-11e8-8d68-419cfa041388\\\\\\"}]\\",
        \\"optionsJSON\\": \\"{\\\\\\"darkTheme\\\\\\":true,\\\\\\"hidePanelTitles\\\\\\":false,\\\\\\"useMargins\\\\\\":true}\\",
        \\"version\\": 1,
        \\"timeRestore\\": false,
        \\"kibanaSavedObjectMeta\\": {
          \\"searchSourceJSON\\": \\"{\\\\\\"query\\\\\\":{\\\\\\"language\\\\\\":\\\\\\"lucene\\\\\\",\\\\\\"query\\\\\\":\\\\\\"\\\\\\"},\\\\\\"filter\\\\\\":[],\\\\\\"highlightAll\\\\\\":true,\\\\\\"version\\\\\\":true}\\"
        }
      }
    },
    \\"config\\": {
      \\"tr-checksum\\": \\"374f4c2661dfcfed4f61defa346cca850436539b\\"
    }
  }
]"
`;
